packages/TorneloScoresheet/src/components/Signature/Signature.tsx

```diff
@@ -57,10 +59,10 @@ const Signature: React.FC<SignatureProps> = ({
             weight={FontWeight.Bold}
             size={40}
             colour={colours.darkenedElements}>
-            {`${fullName(whitePlayer)} \n `}
-            {`${resultText(winnerName, fullName(whitePlayer))}`}
+            {`${fullName(white)} \n `}
+            {`${resultText(winnerName, fullName(white))}`}
```

Comment:
I think it could be good to have whoever is singing have a bold name, and the other player's name is not bold, that way it's extra clear who is supposed to be signing

Reply:
The text is already bold, and I think it's pretty clear who is meant to be signing, if you think it's necessary I can make the 'signature of player' part bigger? It also might be confusing to bold the player in the result part because it's meant to display who won not who is signing.
<img width="196" alt="image" src="https://user-images.githubusercontent.com/64511606/182106639-7501b6b2-b51f-47d8-a10c-acb2d25b79fc.png">

Reply:
hmmm yeh good point actually that might be conbfusing